packages/launchpad/src/components/CodeEditor.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { get } from 'lodash'
+import { ref } from 'vue'
+import CodeEditor from './CodeEditor.vue'
+import Input from './Input/Input.vue'
+
+it('renders', () => {
+  const myCode = ref('console.log("I LOVE formatted code.")')
+
+  cy.mount(() => (<div><CodeEditor
```

Comment:
Might need some form of linting, this is really hard to read

Reply:
Yeah we really need linting.

Reply:
We need prettier